cli.js

```diff
@@ -66,22 +70,27 @@ const onFeedMessage = (buf) => {
 		throw new Error('invalid feed')
 	}
 
+	var results = [];
 	for (const entity of data.entity) {
-		const msg = printAsJSON
+		results.push(printAsJSON || printAsNDJSON
```

Comment:
I'd like to keep the mechanism that, with newline-delimited JSON output, it `JSON.stringify`s and prints immediately.

(The only reason I haven't used a streaming/iterative protobuf parser is that I haven't found one; This would make `print-gtfs-rt-cli` truly compatible with GB-scale data sets.)